.github/CODEOWNERS

```diff
@@ -89,7 +89,7 @@
 
 /modules/programs/mcfly.nix                           @marsam
 
-/modules/programs/mpv.nix                             @tadeokondrak
+/modules/programs/mpv.nix                             @tadeokondrak @thiagokokada
 
```

Comment:
Missed this in #1726, you should also add `/tests/modules/programs/mpv`

Reply:
Done.